tools/audit/repo-mapper/openshift/web.dc.yaml

```diff
@@ -0,0 +1,213 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: repo-mapper
+labels:
+  template: repo-mapper
+objects:
+- apiVersion: v1
+  data:
+    Caddyfile: |
+      :2015
+
+      root * /var/www/html
+
+      file_server
+
+      encode gzip
+      log stdout
+      log stderr
+  kind: ConfigMap
+  metadata:
+    creationTimestamp: null
+    name: caddy-${NAME}-static${SUFFIX}
+    labels:
+      app: ${NAME}-static${SUFFIX}
+      deploymentconfig: ${NAME}-static${SUFFIX}
+      app-name: ${NAME}-static${SUFFIX}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}-static
+    labels: 
+      app: ${NAME}-static${SUFFIX}
+  spec:
+    lookupPolicy:
+      local: false
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}-static${SUFFIX}
+    labels: 
+      app: ${NAME}-static${SUFFIX}
+  spec:
+    minReadySeconds: 20 # should be ready for atleast 20 seconds before the container is considered available. This will allow us
+    # to catch any errors on deploy before they are available to the web
+    replicas: 1
+    selector:
+      deploymentconfig: ${NAME}-static${SUFFIX}
+    strategy:
+      resources:
+        requests:
+          cpu: '150m'
+          memory: '50Mi'
+        limits:
+          cpu: '250m'
+          memory: '105Mi'
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          deploymentconfig: ${NAME}-static${SUFFIX}
+          app: ${NAME}-static${SUFFIX}
+      spec:
+        containers:
```

Comment:
missing service account

Reply:
missed this. those files are no longer need and have been deleted